ui/plugins/controller.js

```diff
@@ -141,6 +147,48 @@ treeherder.controller('PluginCtrl', [
                           }
                           return result;
                         }, []);
+                        if ($scope.jobIds !== null) {
+                            var signatureList = _.keys($scope.jobIds);
+                            var seriesList = [];
+                            $scope.perfJobDetail = [];
+                            $q.all(_.chunk(signatureList, 20).map(function(signatureChunk) {
+                                var url = '/performance/signatures/?signature=' + signatureChunk.pop();
+                                signatureChunk.forEach(function(signature) {
+                                    url = url + '&signature=' + signature;
+                                });
+                                return $http.get(thUrl.getProjectUrl(url, $scope.repoName)).then(
+                                    function(response) {
+                                        _.map(response.data, function(serie, key) {
+                                            serie['signature'] = key;
+                                            seriesList.push(serie);
+                                        })
+                                    }
+                                );
+                            })).then(function(){
+                                var allSubtestSignatures = _.flatten(_.map(seriesList, function(series) {
+                                    return series.subtest_signatures ? series.subtest_signatures : [];
+                                }));
+                                _.forEach(seriesList, function(series) {
+                                    if (!series.subtest_signatures) {
+                                        if (_.contains(allSubtestSignatures, series.signature)) {
+                                            return;
+                                        }
+                                    }
+                                    var detail = {};
+                                    detail['url'] = thServiceDomain + '/perf.html#/graphs?series=[' +
+                                        $scope.repoName+ ',' + series.signature + ',1]&selected=[' +
+                                        $scope.repoName + ',' +  series.signature + ',' +
+                                        $scope.job['result_set_id'] + ',' + $scope.job['id'] + ']';
+                                    detail['value'] = $scope.jobIds[series.signature][0].value;
+                                    detail['suite'] = series.suite;
+                                    if (series.hasOwnProperty('test')) {
+                                        detail['suite'] = series.suite + '_' +
+                                            series.test.toLowerCase();
+                                    }
```

Comment:
To test this, just change your local.conf.js to point to 'stage' instead of 'production'.